ext/native/thin3d/GLRenderManager.cpp

```diff
@@ -806,7 +806,9 @@ void *GLRBuffer::Map(GLBufferStrategy strategy) {
 			if (!hasStorage_) {
 				GLbitfield storageFlags = access & ~(GL_MAP_INVALIDATE_BUFFER_BIT | GL_MAP_FLUSH_EXPLICIT_BIT);
 #ifdef USING_GLES2
+#ifndef GL_EXT_buffer_storage
```

Comment:
This could actually be a bit dangerous - I didn't realize it wasn't defined.

If we define this above `#ifndef GL_GLEXT_PROTOTYPES`, it should prevent gl2ext.h from defining `glBufferStorageEXT` at all, and then we can remove the ifndefs in GLRenderManager and gl3stub.  That should have the least impact on other platforms too.

Sorry, didn't realize before.

-[Unknown]